evm/.example.hardhat.config.js

```diff
@@ -3,7 +3,7 @@ require('@nomicfoundation/hardhat-toolbox');
 
 const env = process.env.ENV || 'testnet';
 const { importNetworks, readJSON } = require('@axelar-network/axelar-contract-deployments');
-const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`@axelar-network/axelar-contract-deployments/axelar-chains-config/info/${env}.json`);
 const keys = readJSON(`${__dirname}/info/keys.json`);
```

Comment:
```suggestion
const keys = readJSON(`${__dirname}/keys.json`);
```
can keep it in the root itself now?